<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="7" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="108583" started-at="2018-08-23T15:57:48Z" finished-at="2018-08-23T15:59:36Z">
    <groups>
    </groups>
    <test name="BranchTeam" duration-ms="108583" started-at="2018-08-23T15:57:48Z" finished-at="2018-08-23T15:59:36Z">
      <class name="com.branch.Tests.TeamInfoTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.branch.Tests.TeamInfoTest@3b6eb2ec]" name="beforeClass" is-config="true" duration-ms="30325" started-at="2018-08-23T08:57:48Z" finished-at="2018-08-23T08:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="employeesDepartmentMatchTest()[pri:0, instance:com.branch.Tests.TeamInfoTest@3b6eb2ec]" name="employeesDepartmentMatchTest" duration-ms="5" started-at="2018-08-23T08:58:18Z" description="Test whether team member&#039;s department assignment match between All tab and each department tabs" finished-at="2018-08-23T08:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- employeesDepartmentMatchTest -->
        <test-method status="PASS" signature="employeesImageMatchTest()[pri:0, instance:com.branch.Tests.TeamInfoTest@3b6eb2ec]" name="employeesImageMatchTest" duration-ms="2" started-at="2018-08-23T08:58:18Z" description="Test whether team member&#039;s images match between All tab and each department tabs" finished-at="2018-08-23T08:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- employeesImageMatchTest -->
        <test-method status="PASS" signature="employeesNameMatchTest()[pri:0, instance:com.branch.Tests.TeamInfoTest@3b6eb2ec]" name="employeesNameMatchTest" duration-ms="1" started-at="2018-08-23T08:58:18Z" description="Test whether team member&#039;s name match between All tab and each department tabs" finished-at="2018-08-23T08:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- employeesNameMatchTest -->
        <test-method status="PASS" signature="profileLinkMatchTest()[pri:0, instance:com.branch.Tests.TeamInfoTest@3b6eb2ec]" name="profileLinkMatchTest" duration-ms="2" started-at="2018-08-23T08:58:18Z" description="Test whether team member&#039;s profile link match between All tab and each department tabs" finished-at="2018-08-23T08:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- profileLinkMatchTest -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.branch.Tests.TeamInfoTest@3b6eb2ec]" name="afterClass" is-config="true" duration-ms="175" started-at="2018-08-23T08:58:18Z" finished-at="2018-08-23T08:58:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.branch.Tests.TeamInfoTest -->
      <class name="com.branch.Tests.TeamMembersCountMatchTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.branch.Tests.TeamMembersCountMatchTest@1e643faf]" name="beforeClass" is-config="true" duration-ms="612" started-at="2018-08-23T08:58:18Z" finished-at="2018-08-23T08:58:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="employeesCountTest(org.openqa.selenium.Dimension)[pri:0, instance:com.branch.Tests.TeamMembersCountMatchTest@1e643faf]" name="employeesCountTest" duration-ms="29118" started-at="2018-08-23T08:58:19Z" description="Test whether team members count match between All tab and each department tabs" data-provider="screenDimension" finished-at="2018-08-23T08:58:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[(480, 800)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Team members count mismatch between All tabs and Sum of other tabs expected [131] but found [126]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Team members count mismatch between All tabs and Sum of other tabs expected [131] but found [126]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at com.branch.Tests.TeamMembersCountMatchTest.employeesCountTest(TeamMembersCountMatchTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- employeesCountTest -->
        <test-method status="FAIL" signature="employeesCountTest(org.openqa.selenium.Dimension)[pri:0, instance:com.branch.Tests.TeamMembersCountMatchTest@1e643faf]" name="employeesCountTest" duration-ms="24779" started-at="2018-08-23T08:58:48Z" description="Test whether team members count match between All tab and each department tabs" data-provider="screenDimension" finished-at="2018-08-23T08:59:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[(850, 800)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Team members count mismatch between All tabs and Sum of other tabs expected [131] but found [126]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Team members count mismatch between All tabs and Sum of other tabs expected [131] but found [126]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at com.branch.Tests.TeamMembersCountMatchTest.employeesCountTest(TeamMembersCountMatchTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- employeesCountTest -->
        <test-method status="FAIL" signature="employeesCountTest(org.openqa.selenium.Dimension)[pri:0, instance:com.branch.Tests.TeamMembersCountMatchTest@1e643faf]" name="employeesCountTest" duration-ms="23375" started-at="2018-08-23T08:59:13Z" description="Test whether team members count match between All tab and each department tabs" data-provider="screenDimension" finished-at="2018-08-23T08:59:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[(1200, 800)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Team members count mismatch between All tabs and Sum of other tabs expected [131] but found [126]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Team members count mismatch between All tabs and Sum of other tabs expected [131] but found [126]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at com.branch.Tests.TeamMembersCountMatchTest.employeesCountTest(TeamMembersCountMatchTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- employeesCountTest -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.branch.Tests.TeamMembersCountMatchTest@1e643faf]" name="afterClass" is-config="true" duration-ms="173" started-at="2018-08-23T08:59:36Z" finished-at="2018-08-23T08:59:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.branch.Tests.TeamMembersCountMatchTest -->
    </test> <!-- BranchTeam -->
  </suite> <!-- Suite -->
</testng-results>
